grafana-plugin/src/pages/settings/tabs/ChatOps/tabs/SlackSettings/SlackSettings.tsx

```diff
@@ -124,8 +124,30 @@ class SlackSettings extends Component<SlackProps, SlackState> {
                 </WithPermissionControl>
               </Field>
             </HorizontalGroup>
-            <WithPermissionControl userAction={UserActions.ChatOpsWrite}>
-              <WithConfirm title="Are you sure to delete this Slack Integration?">
+            <WithPermissionControl userAction={UserActions.ChatOpsUpdateSettings}>
```

Comment:
`UserActions.ChatOpsUpdateSettings` is the correct permission here according to the endpoint that is called by `this.removeSlackIntegration` (the `onClick` handler)